File: Data Structures & Algorithms/subarray-sum-equals-k/submission-0.cpp
class Solution {
public:
    int subarraySum(vector<int>& nums, int k) {
        int count = 0;

        for(int i = 0; i < nums.size(); ++i) {
            if (k - nums[i] == 0) {
                count++;
            }

            int remain = k - nums[i];
            for(int j = i+1; j<nums.size(); ++j) {
                if(remain - nums[j] == 0) {
                    count++;
                }
                remain -= nums[j];
            }
            
        }
        return count;
    }
};

File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """

        count = [0] * 3

        for n in nums:
            if n == 0:
                count[0] += 1
            if n == 1:
                count[1] += 1
            if n == 2:
                count[2] += 1
        
        index = 0
        for i in range(3):
            while count[i]:
                count[i] -= 1
                nums[index] = i
                index += 1

        
        